css-grid-1/issues-cr-2016.html

```diff
@@ -292,7 +292,7 @@ <h1>CSS Grid Layout Level 1 Disposition of Comments for 2016-09-29 CR</h1>
 <span>Resolved: <a href='https://lists.w3.org/Archives/Public/www-style/2017Apr/0018.html'>https://lists.w3.org/Archives/Public/www-style/2017Apr/0018.html</a> <a href="https://www.w3.org/Mail/flatten/index?subject=https%3A%2F%2Flists.w3.org%2FArchives%2FPublic%2Fwww-style/2017Apr/0018.html"><abbr title="Thread">&#x2208;</abbr></a></span></pre>
 <pre class=' a' id='issue-32'>
 <span>Issue 32. <a href='#issue-32'>#</a></span>
-<span>Summary:  Computed value for single value &lt;flex>?</span>
+<span>Summary:  Computed value for single value &lt;fraction>?</span>
```

Comment:
Don't change descriptions of past discussions!

Reply:
Search & replace mistake. Thanks for noticing! Fixed now.